### ISSUE TITLE
Initial Technical Assessment

### ISSUE THREAD

**Issue by eddietejeda** [18F/performance.gov#28]
- [x] Review archived Drupal data dump from previous system
- [ ] Conduct basic assessment/survey of existing [agency performance backend systems](https://drive.google.com/drive/folders/0B2stDO5hLAHkZUxPYy04X3F3dk0) work
    - [x] [BFEM](https://arc.publicdebt.treas.gov/financial_management_budget_formulation.htm) (Treasury’s PMM Budget formulation system)
    - [x] FACTSInfo
    - [x] BAS/ new BFS
    - [x] IBM Cognos
    - [x] Departmental e-Business Suite (DEBS)
    - [x] Oracle BI 
    - [ ] Microsoft Access
    - [ ] MS Word, MS Powerpoint
    - [ ] Google site with spreadsheets and presentation decks
    - [x] Program Performance Tracking System     
    - [x] Others?

**Comment by adborden** [comment]
@eddietejeda I'm thinking I will skip over the office tools like Access, Word, and Google docs. I don't think there's much to analyze there.

**Comment by eddietejeda** [comment]
For now, sure. Although, at some point, we'll have to see how they've structured data. MS Access might actually be interesting.

**Comment by AminPIC** [comment]
@eddietejeda I noticed that you have check marks next to BFEM and DEBS. Does that mean you have already done a basic assessment of these systems or does it mean those are outstanding?

**Comment by eddietejeda** [comment]
To be perfectly honest, I don't remember checking off those boxes... and if I did, why I did it! 

But it should be for when we've done some assessment of it.

**Comment by adborden** [comment]
That was me who checked off the boxes. You can see the work in progress [tool assessment](https://github.com/18F/performance.gov/blob/master/performance-tools.md) and [survey assessment](https://docs.google.com/a/gsa.gov/document/d/18o4zol2NEGDHEecQ5_Wx9JmANQrY4bU2zIFtFCNw_vo/edit?usp=sharing) . Are we looking for something more detailed? I interpreted this story to be a basic understanding based on the information we've already collected.

**Comment by adborden** [comment]
I created a slimmer version of the db dump https://drive.google.com/file/d/1iHpOWyKBOeYyZd9q9MO6JudIRRdbUs7o/view

**Comment by adborden** [comment]
Waiting on a meeting with Adam L about GPRA concepts for the data model.
